ee/clickhouse/queries/event_query.py

```diff
@@ -166,13 +166,15 @@ def _get_prop_groups(self, prop_group: Optional[PropertyGroup]) -> Tuple[str, Di
         if not prop_group:
             return "", {}
 
+        outer_properties = self._column_optimizer.property_optimizer.parse_property_groups(prop_group).outer
+
         return parse_prop_grouped_clauses(
             team_id=self._team_id,
-            property_group=prop_group,
+            property_group=outer_properties,
             prepend="global",
             table_name=self.EVENT_TABLE_ALIAS,
             allow_denormalized_props=True,
-            person_properties_mode=PersonPropertiesMode.EXCLUDE,
+            person_properties_mode=PersonPropertiesMode.USING_PERSON_PROPERTIES_COLUMN,
```

Comment:
@neilkakkar i think you're right "exclude" mode doesn't really make sense anymore. I changed this one to `USING_PERSON_PROPERTIES_COLUMN` so that the outer query filters will use properties from the person subquery and not do a costly distinct_id IN.

TODO: to a complete cleanse of .exclude/replacement everywhere else

Reply:
I removed most of these instances, except one, which will go out in the follow up PR, as it's touching lots of entity.properties